examples/ERP/plot_embedding_EEG.py

```diff
@@ -0,0 +1,68 @@
+"""
+====================================================================
+Embedding ERP EEG data in 2D Euclidean space with Laplacian Eigenmaps
+====================================================================
+
+Spectral embedding via Laplacian Eigenmaps of a set of ERP data.
+
+"""
+# Authors: Pedro Rodrigues <pedro.rodrigues01@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from pyriemann.estimation import ERPCovariances
+from pyriemann.embedding import Embedding
+
+import mne
+from mne import io
+from mne.datasets import sample
+
+from matplotlib import pyplot as plt
+
+print(__doc__)
+
+data_path = sample.data_path()
+
+###############################################################################
+# Set parameters and read data
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+tmin, tmax = -0., 1
+event_id = dict(vis_l=3, vis_r=4)
+
+# Setup for reading the raw data
+raw = io.Raw(raw_fname, preload=True, verbose=False)
+raw.filter(2, None, method='iir')  # replace baselining with high-pass
+events = mne.read_events(event_fname)
+
+raw.info['bads'] = ['MEG 2443']  # set bad channels
+picks = mne.pick_types(raw.info, meg=False, eeg=True, stim=False, eog=False,
+                       exclude='bads')
+
+# Read epochs
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=False,
+                    picks=picks, baseline=None, preload=True, verbose=False)
+
+X = epochs.get_data()
+y = epochs.events[:, -1]
+
+###############################################################################
+# Embedding the Xdawn covariance matrices with Diffusion maps
+
+covs = ERPCovariances(estimator='oas', classes=[3, 4]).fit_transform(X, y)
```

Comment:
ERPCov is also Supervised. 
Using XdawnCov was fine (and probably faster), just cross-validates the estimation i.e

```
train = range(0, len(y), 2)
test = range(1, len(y), 2)
covs = XdawnCov().fit(X[train], y[train]).transform(X[test])
```